xrmocap/ops/bottom_up_association/associate/fourdag_associate.py

```diff
@@ -0,0 +1,956 @@
+# yapf: disable
```

Comment:
As offline discussion, we may rename the folder to `graph_solver` and split the following class to two classes, namely `build_graph` and `associate_graph`